spec/features/users/sign_in_spec.rb

```diff
@@ -131,14 +131,24 @@
         expect(page).to_not have_content(t('forms.buttons.submit.continue'))
 
         fill_in_credentials_and_click_sign_in(user.email, user.password)
-
         expect(page).to have_content t('errors.invalid_authenticity_token')
 
         fill_in_credentials_and_click_sign_in(user.email, user.password)
-
         expect(current_path).to eq user_two_factor_authentication_path
       end
     end
+
+    xit 'displays the session timeout modal, does not allow the user to submit', js: true do
```

Comment:
note that this is pending -- it passes even without the js above. 

I think this is because `fill_in` does not actually focus on the field (which is what is required for "enter" to submit the form).

Any ideas on how to get this test to actually test what we want it to?